luigi_pipeline/lib/model/mito_mt_schema.py

```diff
@@ -0,0 +1,183 @@
+import hail as hl
+
+from lib.model.base_mt_schema import row_annotation
+from lib.model.seqr_mt_schema import SeqrVariantSchema, SeqrGenotypesSchema
+
+
+class SeqrMitoVariantSchema(SeqrVariantSchema):
+
+    def __init__(self, *args, ref_data, high_constraint_region, **kwargs):
+        super().__init__(*args, ref_data=ref_data, clinvar_data=None, **kwargs)
+        self._high_constraint_region = high_constraint_region
+
+    # Mitochondrial only fields
+    @row_annotation()
+    def gnomad(self):
+        return self._selected_ref_data.gnomad_mito
+
+    @row_annotation()
+    def mitomap(self):
+        return self._selected_ref_data.mitomap
+
+    @row_annotation(name='APOGEE')
+    def mitimpact(self):
+        return self._selected_ref_data.mitimpact
+
+    @row_annotation(name='HmtVar')
+    def hmtvar(self):
+        return self._selected_ref_data.hmtvar
+
+    @row_annotation()
+    def helix(self):
+        return self._selected_ref_data.helix_mito
+
+    @row_annotation()
+    def common_low_heteroplasmy(self):
+        return self.mt.common_low_heteroplasmy
+
+    @row_annotation()
+    def hap_defining_variant(self):
+        return self.mt.hap_defining_variant
+
+    @row_annotation()
+    def mitotip_trna_prediction(self):
+        return self.mt.mitotip_trna_prediction
+
+    @row_annotation()
+    def high_constraint_region(self):
+        return hl.is_defined(self._high_constraint_region[self.mt.locus])
+
+    # Fields with the same name but annotated differently
+    @row_annotation()
+    def clinvar(self):
+        return self._selected_ref_data.clinvar_mito
+
+    @row_annotation()
+    def dbnsfp(self):
+        return self._selected_ref_data.dbnsfp_mito
+
+    @row_annotation(name='AC')
+    def ac(self):
+        return self.mt.AC_hom
+
+    @row_annotation(name='AF')
+    def af(self):
+        return self.mt.AF_hom
+
+    @row_annotation(name='AN')
+    def an(self):
+        return self.mt.AN
+
+    @row_annotation(name='AC_het')
+    def ac_het(self):
+        return self.mt.AC_het
+
+    @row_annotation(name='AF_het')
+    def af_het(self):
+        return self.mt.AF_het
+
+    # Remove the inherited unwanted annotation function
```

Comment:
this is not a good approach for only inheriting some fields from a base class - its very hard to read and very hard to make sure new fields get added to the appropriate classes. Please go with the approach I originally asked for with a `SeqrCoreSchema`  containing only the overlapping fields that is used both here and by `SeqrVariantSchema`